autoscale/src/main/java/com/sequenceiq/periscope/domain/ScalingTrigger.java

```diff
@@ -0,0 +1,118 @@
+package com.sequenceiq.periscope.domain;
+
+import java.util.Date;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.persistence.SequenceGenerator;
+import javax.persistence.Table;
+import javax.persistence.Temporal;
+import javax.persistence.TemporalType;
+
+import com.sequenceiq.periscope.api.model.TriggerStatus;
+
+@Entity
+@Table(name = "scaling_trigger")
+public class ScalingTrigger implements Clustered {
```

Comment:
A few potential enhancements: 
1. Track the actual scaling operation - e.g. START 5 nodes, DELETE {node1, node2..}
2. Any way we can link this to a trigger rule - i.e. which rule resulted in this trigger (not sure how valuable this is right now, since the ids there are not very stable iirc)

Reply:
>Track the actual scaling operation - e.g. START 5 nodes, DELETE {node1, node2..}

This message will get populated into the `trigger_reason` field, i.e, yarnUpscale count and yarnDecommission recommendations

>Any way we can link this to a trigger rule - i.e. which rule resulted in this trigger (not sure how valuable this is right now, since the ids there are not very stable iirc)
Does this refer to the autoscale policy that resulted in the trigger? I can add this as part of an improvement jira